applications/frontend/src/app/components/controlpanel/ControlPanelGrid.tsx

```diff
@@ -207,6 +212,13 @@ export const ControlPanelGrid: FunctionComponent<{ globalState: GlobalState }> =
 		[socket]
 	)
 
+	const setAndEmitGlobalSettingsTrigger = useCallback(
+		(threshold: number[]) => {
+			socket?.emit(GLOBAL_SETTINGS_UPDATE, { ...globalState, crowdThreshold: threshold })
```

Comment:
This doesn't look right. Partial GlobalSettingsState should suffice. No need to copy whole globalState (which is in the wrong place here anyway)